src/Apps/Ingress/Components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback } from 'react';
+import {
+    IHomePageProps,
+    IHomePageStyleProps,
+    IHomePageStyles
+} from './HomePage.types';
+import { getStyles } from './HomePage.styles';
+import {
+    BaseButton,
+    classNamesFunction,
+    Pivot,
+    PivotItem,
+    Stack,
+    styled
+} from '@fluentui/react';
+import { useExtendedTheme } from '../../../../Models/Hooks/useExtendedTheme';
+import {
+    INGRESS_TRANSLATION_COMMON_PATH,
+    INGRESS_TRANSLATION_PATH
+} from '../../Models/Constants';
+import { useNavigationContext } from '../../Models/Context/NavigationContext';
+import {
+    NavigationContextActionType,
+    PageNames
+} from '../../Models/Context/NavigationContext.types';
+import { useTranslation } from 'react-i18next';
+
+const getClassNames = classNamesFunction<
+    IHomePageStyleProps,
+    IHomePageStyles
+>();
+
+const TRANSLATE_KEY_ROOT = `${INGRESS_TRANSLATION_PATH}.IngressHomePage`;
```

Comment:
do we need to put the INGRESS_TRANSLATION_PATH into constants file? It seems it is not used anywhere else, so can we just put it here hardcoded as we do in other components?

Reply:
I thought this would be a good pattern for components later, we usually have a loc object for a large component, like advanced search or an app like OAT, and this could be a way to have the same base for all Ingress components.